src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -2285,6 +2286,20 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 break;
             }
 
+            case NI_Sve_LoadVectorFirstFaulting:
+            {
+                insScalableOpts sopt = (opt == INS_OPTS_SCALABLE_B) ? INS_SCALABLE_OPTS_NONE : INS_SCALABLE_OPTS_LSL_N;
+                GetEmitter()->emitIns_R_R_R_R(ins, emitSize, targetReg, op1Reg, op2Reg, REG_ZR, opt, sopt);
+                break;
+            }
+
+            case NI_Sve_SetFfr:
+            {
+                assert(targetReg == REG_NA);
+                GetEmitter()->emitIns_R(ins, emitSize, op1Reg, opt);
```

Comment:
Don't we need special codegen for `GetFfr` too otherwise it will generate [RDFFR (predicated)](https://docsmirror.github.io/A64/2023-06/rdffr_p_p_f.html) instead of [RDFFR (unpredicated)](https://docsmirror.github.io/A64/2023-06/rdffr_p_f.html)?

Reply:
Yea, we can add an optimization to set SETFFR if op1 is contained and IsAllBitsSet. I will add the opt.

Reply:
No, this comment is specifically for `GetFfr` and not for `SetFfr`.

Reply:
Ah, gotcha. I mis-read.